transport/x/grpc/response_writer.go

```diff
@@ -54,7 +53,7 @@ func (r *responseWriter) AddHeaders(headers transport.Headers) {
 }
 
 func (r *responseWriter) SetApplicationError() {
-	r.isApplicationError = true
+	r.AddSystemHeader(ApplicationErrorHeader, "1")
```

Comment:
Let’s use the `"error"` constant here, like HTTP https://sourcegraph.com/github.com/yarpc/yarpc-go@33f018e504b97d682f0d62e5359f4b16484a9e94/-/blob/transport/http/constants.go#L79:2

The value isn’t all that important, and its presence should be enough on the read side, but it’s easier to understand on the wire if we spell it out.

Reply:
Changing